BUILD.gn

```diff
@@ -4179,6 +4179,10 @@ v8_component("v8_libbase") {
     sources += [ "src/base/ubsan.cc" ]
   }
 
+  if(v8_current_cpu == "riscv" || v8_current_cpu == "riscv64"){
+    libs += [ "atomic" ]
+  }
+
```

Comment:
How is this related to skip online trampoline?

Reply:
Building native neeads it